### PR TITLE
[netatmo] Adding the ability to append a postfix to registered webhook.

### PR DESCRIPTION
**Pull request by clinique**
The problem was registering a webhook like `https://mydomain:443/netatmo/webhook/5xxxxxddddggggg47d3` was really difficult to address behind a reverse proxy relaying the payload (one day of effort tuning nginx without success). With this extra parameter it's easy to register  `https://mydomain:443/netatmo/webhook/5xxxxxddddggggg47d3/` or  `https://mydomain:443/netatmo/webhook/5xxxxxddddggggg47d3/webhook.php`  

Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.netatmo/README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The Account bridge has the following configuration elements:
 | clientId          | String | Yes      | Client ID provided for the application you created on http://dev.netatmo.com/createapp                                 |
 | clientSecret      | String | Yes      | Client Secret provided for the application you created                                                                 |
 | webHookUrl        | String | No       | Protocol, public IP and port to access openHAB server from Internet                                                    |
+| webHookPostfix    | String | No       | String appended to the generated webhook address                                                                       |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.netatmo/README.md:51]: Maybe you could mention in the description (here and in the i18n property) that the piostfix must start with a /

**Comment by lolodomo** [comment]
And you have a conflict on README to resolve.

File: bundles/org.openhab.binding.netatmo/README.md (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md:51]: Seemed obvious, but you're correct, it's better explained.

**Comment by clinique** [comment]
> And you have a conflict on README to resolve.

This is weird, the conflict is on my modification... :-/

File: bundles/org.openhab.binding.netatmo/README.md (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.netatmo/README.md]: Not so obvious as I will have used naturally a / while you say now it must be a \

Comment (lolodomo) [reply bundles/org.openhab.binding.netatmo/README.md]: Are you sure of \ ? That looks strange for  HTTP URL.

File: bundles/org.openhab.binding.netatmo/README.md [hunk 2]
```diff
@@ -157,7 +158,7 @@ If you did not manually create things in the *.things file, the Netatmo Binding
 ## Channels
 
 ### Weather Station Main Indoor Device
-
+1
```
Comment (lolodomo) [inline bundles/org.openhab.binding.netatmo/README.md:161]: Why adding that ?

File: bundles/org.openhab.binding.netatmo/README.md [hunk 3]
```diff
@@ -106,8 +107,8 @@ http(s)://xx.yy.zz.ww:443
 
 Your Netatmo App will be configured automatically by the bridge to the endpoint:
 
-```text
-http(s)://xx.yy.zz.ww:443/netatmo/webhook/<_CLIENT_ID_>
+```
```
Comment (lolodomo) [inline bundles/org.openhab.binding.netatmo/README.md:110]: The "text" was there probably for a good reason.
By the way, there is one just above for another HTTP URL. It should be maintained consistent, either remove both or keep both.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.netatmo/README.md (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md]: Getting too old I guess. Good luck you're there as the door keeper. Happy new year @lolodomo

File: bundles/org.openhab.binding.netatmo/README.md (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md]: Weird, I do not see this in my code

File: bundles/org.openhab.binding.netatmo/README.md (hunk 3 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md:116]: I don't have that neither

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.netatmo/README.md (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.netatmo/README.md]: Happy new year too.

**Comment by jlaur** [comment]
@lolodomo - you merged this into the hotfix branch, was this intended?

**Comment by lolodomo** [comment]
No sorry I did not see it was provided in the wrong branch.

**Comment by clinique** [comment]
> No sorry I did not see it was provided in the wrong branch.

I just did report it also in main branch.

**Comment by jlaur** [comment]
> No sorry I did not see it was provided in the wrong branch.

No problem, I have reverted it.

**Pull request merged**